src/Tasks/loadTasks.php

```diff
@@ -12,53 +12,61 @@
 
 trait loadTasks
 {
-    use metricsTasks;
+	use metricsTasks;
 
-    /**
-     * Map Task
-     *
-     * @param   String  $target  - The target directory
-     *
-     * @return  Map
-     */
-    protected function taskMap($target)
-    {
-        return new Map($target);
-    }
+	/**
+	 * Map Task
+	 *
+	 * @param   String  $target  - The target directory
+	 *
+	 * @return  Map
```

Comment:
And the other thing is if the class is in the same namespace as the file, you don't need the use statement.  PHP assumes if it's not a fully qualified class name or prefixed with the global namespace `\` that it's a class in the same namespace and IDEs and doc generators behave the same way.